CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [v0.28.0](https://github.com/56quarters/cadence/tree/0.28.0) - 2022-01-19
```

Comment:
This is great work, you should add your GitHub handle here for credit for this feature!

Reply:
Could you also add a section to both the `lib.rs` of the `cadence` crate and `README.md` of the `cadence` crate with an example of how this is used and an explanation of why it's useful? Maybe a short code example in the `examples/` directory.

Reply:
Thanks for the review, I'll work on making the requested changes!